modules/openapi-generator/src/main/resources/python/python-experimental/api_client.mustache

```diff
@@ -517,19 +522,34 @@ class ApiClient(object):
         else:
             return content_types[0]
 
-    def update_params_for_auth(self, headers, querys, auth_settings):
+    def update_params_for_auth(self, headers, querys, auth_settings,
+                               resource_path=None, method=None, body=None):
```

Comment:
Aren't resource_path and method required?
If so please make them positional arguments without setting them to None by default.